validator/lib/validateTileset.js

```diff
@@ -98,16 +154,25 @@ function validateTileHierarchy(root, tilesetDirectory) {
             var tileSphere = tile.boundingVolume.sphere;
 
             if (defined(contentRegion) && defined(tileRegion) && !regionInsideRegion(contentRegion, tileRegion)) {
+                var string_json = JSON.stringify(tile.content, null, 4);
+                console.log('Error in tile:');
+                console.log(string_json);
                 return 'content region [' + contentRegion + '] is not within tile region + [' + tileRegion + ']';
             }
 
             if (defined(contentSphere) && defined(tileSphere) && !sphereInsideSphere(contentSphere, tileSphere)) {
+                var string_json = JSON.stringify(tile.content, null, 4);
+                console.log('Error in tile:');
+                console.log(string_json);
                 return 'content sphere [' + contentSphere + '] is not within tile sphere + [' + tileSphere + ']';
             }
         }
 
         if (defined(tile.refine)) {
             if (tile.refine !== 'ADD' && tile.refine !== 'REPLACE') {
+                var string_json = JSON.stringify(tile, null, 4);
+                console.log('Error in tile:');
+                console.log(string_json);
```

Comment:
Create a helper function for all of these. It will take the tile and the error message (e.g. `'Refine property in tile must have either "ADD" or "REPLACE" as its value.'`) and return an error message with the string json appended.